### PR TITLE
Refactor/chain/edit

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: webapp/views/App/views/Analysis/Chain/store/actions/rChain.js [hunk 1]
```diff
@@ -34,3 +34,6 @@ export const openRChain = () => async (dispatch, getState) => {
   // Open RStudio in a new page
   window.open(_getRStudioUrl(), 'rstudio')
 }
+
+export const openRChain = () => async (dispatch, getState) => {}
+*/
```
Comment (ramirobg94) [inline webapp/views/App/views/Analysis/Chain/store/actions/rChain.js:39]: this file need to be refactorized

File: webapp/store/survey/surveyInfo/reducer.js [hunk 2]
```diff
@@ -5,11 +5,6 @@ import * as Survey from '@core/survey/survey'
 // App actions
 import { SystemActions } from '@webapp/store/system'
 
-// Processing chain actions
-import { chainSave, chainDelete } from '@webapp/loggedin/modules/analysis/chain/actions'
-import { stepDelete } from '@webapp/loggedin/modules/analysis/step/actions'
-import { calculationDelete } from '@webapp/loggedin/modules/analysis/calculation/actions'
-
 import * as SurveyActions from '../actions'
```
Comment (minotogna) [inline webapp/store/survey/surveyInfo/reducer.js:8]: when saving chain, do we set the survey as draft?
if not,  in saveChain action we should ```dispatch(SurveyActions.markSurveyDraft())```

Comment (ramirobg94) [reply webapp/store/survey/surveyInfo/reducer.js:8]: i have to fix that ahhaha

File: webapp/views/App/views/Analysis/Analysis.js [hunk 3]
```diff
@@ -9,22 +8,18 @@ import CategoriesView from '@webapp/loggedin/surveyViews/categories/categoriesVi
...
 
 const Analysis = () => {
-  const dispatch = useDispatch()
   const history = useHistory()
   return (
     <SurveyDefsLoader
       draft
       validate={false}
       requirePublish
-      onSurveyCycleUpdate={() => dispatch(navigateToChainsView(history))}
```
Comment (SteRiccio) [inline webapp/views/App/views/Analysis/Analysis.js:27]: I prefer the old way to navigate to chain views...!

File: webapp/store/survey/surveyInfo/reducer.js (hunk 2 above)
Comment (minotogna) [reply webapp/store/survey/surveyInfo/reducer.js:8]: this branch or another one?

Comment (ramirobg94) [reply webapp/store/survey/surveyInfo/reducer.js:8]: other branch

**Pull request merged**